nixos/modules/services/misc/ombi.nix

```diff
@@ -0,0 +1,76 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let cfg = config.services.ombi;
+
+in {
+  options = {
+    services.ombi = {
+      enable = mkEnableOption "Ombi";
+
+      dataDir = mkOption {
+        type = types.str;
+        default = "/var/lib/ombi/.config/Ombi";
```

Comment:
`/var/lib/ombi` seems like a more reasonable default, no?

Reply:
I tried to follow the same scheme from Sonarr, Radarr and Jackket, but I do agree that it makes more sense like that.